eng/Versions.props

```diff
@@ -255,6 +255,11 @@
     -->
     <NewtonsoftJsonVersion>12.0.2</NewtonsoftJsonVersion>
     <StreamJsonRpcVersion>2.6.104</StreamJsonRpcVersion>
+    <MicrosoftBclAsyncInterfacesVersion>1.1.1</MicrosoftBclAsyncInterfacesVersion>
```

Comment:
Is this intentional? There's a duplicate entry below for `MicrosoftBclAsyncInterfacesVersion`